src/table/table.scss

```diff
@@ -191,9 +191,10 @@
       }
     }
 
+    @include iui-transition-group;
```

Comment:
Maybe move to the top that it would be easier to see.

Reply:
moved somewhere